wallet/wallet.go

```diff
@@ -2382,6 +2382,88 @@ func (w *Wallet) TransactionSummary(txHash *chainhash.Hash) (*TransactionSummary
 	return txSummary, err
 }
 
+// GetTicketsResult response struct for gettickets rpc request
+type GetTicketsResult struct {
+	Tickets []*TicketSummary
+}
+
+// GetTickets implements the rpc request command for gettickets
```

Comment:
This comment really needs to be more specific regarding its behavior and what all parameters do, and we shouldn't isolate this API as only for the RPC server.